### ISSUE TITLE
Installation fails due to Parsedown version conflict with Doxter

### ISSUE THREAD

**Issue by chadcrowell** [TopShelfCraft/Wordsmith#22]
Possible to change your Parsedown requirement to match his?

```
Composer output: Package "craftcms/vue-asset" listed for update is not installed. Ignoring.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Problem 1
- selvinortiz/doxter 3.4.7 requires erusev/parsedown-extra 0.8.0-beta-1 -> satisfiable by erusev/parsedown-extra[0.8.0-beta-1].
- selvinortiz/doxter 3.4.7 requires erusev/parsedown-extra 0.8.0-beta-1 -> satisfiable by erusev/parsedown-extra[0.8.0-beta-1].
- selvinortiz/doxter 3.4.7 requires erusev/parsedown-extra 0.8.0-beta-1 -> satisfiable by erusev/parsedown-extra[0.8.0-beta-1].
- Can only install one of: erusev/parsedown-extra[0.7.1, 0.8.0-beta-1].
- Can only install one of: erusev/parsedown-extra[0.7.1, 0.8.0-beta-1].
- Can only install one of: erusev/parsedown-extra[0.7.1, 0.8.0-beta-1].
- topshelfcraft/wordsmith 3.0.4 requires erusev/parsedown-extra ^0.7.1 -> satisfiable by erusev/parsedown-extra[0.7.1].
- Installation request for topshelfcraft/wordsmith 3.0.4 -> satisfiable by topshelfcraft/wordsmith[3.0.4].
- Installation request for selvinortiz/doxter (locked at 3.4.7, required as ^3.4) -> satisfiable by selvinortiz/doxter[3.4.7].

Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.
```

**Comment by michaelrog** [comment]
(c.f. https://github.com/selvinortiz/craft-plugin-doxter)

**Comment by michaelrog** [comment]
[`erusev/parsedown`](https://github.com/erusev/parsedown) is only tested through 7.3. 🤔 I'm hesitant to change the stability requirement, unless there's significant functionality we're missing out on by not requiring the newest beta version... But I'll look into it. Definitely do want Doxter and Wordsmith to be usable together. 

@selvinortiz, what was your reason for upgrading [Doxter](https://github.com/selvinortiz/craft-plugin-doxter)'s Parsedown to 0.8 before the stable release?

**Comment by selvinortiz** [comment]
@michaelrog The latest "stable" release of Parsedown breaks footnotes (https://github.com/erusev/parsedown-extra/issues/118) to patch a security issue in previous versions.

I wanted footnotes to work but I did not want to go back to a version that could introduce potential security issues. I opted instead, to test (at great length) the beta version that addressed the security concerns and fixeed footnotes.
